extras/AFC_prep.py

```diff
@@ -94,7 +95,13 @@ def PREP(self, gcmd):
                 if 'color' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['color']='#000000'
                 if 'weight' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['weight'] = 0
                 if 'runout_lane' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['runout_lane']='NONE'
-                if 'map' not in self.AFC.lanes[LANE.unit][LANE.name] or self.AFC.lanes[LANE.unit][LANE.name]['map'] is None: self.AFC.lanes[LANE.unit][LANE.name]['map'] = LANE.map
+                if 'map' not in self.AFC.lanes[LANE.unit][LANE.name]:
```

Comment:
Should probably keep `or self.AFC.lanes[LANE.unit][LANE.name]['map'] is None:` check in just in case someones config file gets map set to null. This will cover the null cases
```suggestion
                if 'map' not in self.AFC.lanes[LANE.unit][LANE.name] or self.AFC.lanes[LANE.unit][LANE.name]['map'] is None::
```

Reply:
the config input in the stepper makes it 'NONE' if null

Reply:
But if for some weird reason this `"map": null` gets stuck in someones AFC.var.unit file it will error out. I agree it should not happen but its always the cases we don't account for that breaks things

Reply:
added back in